test/integration/dashboard/student_can_see_events_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class User::NewEventsTest < ActionDispatch::IntegrationTest
+  test 'a student can see a single event that is displayed' do 
+    FactoryGirl.create(:user, :default)
+    event = Event.create(start_date: "12-04-14",
+                         start_time: "1:00 PM",
+                         end_date: "12-05-14",
+                         end_time: "5:00 PM",
+                         name: "Museum Outing",
+                         description: "Museum of Nature and Science",
+                         location: "LV Museum"
+                            )
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.